### ISSUE TITLE
Mike BO Memory Dump Files

### ISSUE THREAD

**Issue by huan** [juzibot/donut-tester#25]
### Jun 22

1. [mike-bo-dump-20200622.tar.gz](https://github.com/juzibot/donut-tester/files/4811108/mike-bo-dump-20200622.tar.gz)

**Comment by su-chang** [comment]
Thank you very much for the dump files.

Could you please also provided some logs about your bot when WeChat dump?

BTW, how about let Mike Bot work in docker environment? If you are interesting about it, I will put your `pub_key` in `authorized_keys`.

**Comment by huan** [comment]
I did not save the logs for mike, will try to save them in the future.

I'm interested in the docker environment, please put my key in, thanks!

**Comment by huan** [comment]
### Jun 23

`@juzibot/wechaty-puppet-donut@0.3.63`

1. [mike-bo-dump-20200623.tar.gz](https://github.com/juzibot/donut-tester/files/4814471/mike-bo-dump-20200623.tar.gz)

**Comment by huan** [comment]
### Jun 27

`wechaty-puppet-donut@0.3.66`

[mike.tar.gz](https://github.com/juzibot/donut-tester/files/4840273/mike.tar.gz)

**Comment by huan** [comment]
### Jun 30

0.3.66

[mike.tar.gz](https://github.com/juzibot/donut-tester/files/4851507/mike.tar.gz)

**Comment by huan** [comment]
### Jun 2

0.3.74

[mike-20200702.tar.gz](https://github.com/juzibot/donut-tester/files/4861872/mike-20200702.tar.gz)

**Comment by huan** [comment]
Are those dump files helpful?

I hope we have found something from those dump files. @su-chang

**Comment by su-chang** [comment]
Yes, 小邪 are working on it. Thank you very much for the dump file.

Could you please put the related donut log into the zip file? Maybe we can also find something in it.

**Comment by huan** [comment]
Great to know they are useful.

Will try to attach the log next time.

**Comment by huan** [comment]
### Jun 3

[mike.20200703.tar.gz](https://github.com/juzibot/donut-tester/files/4874713/mike.20200703.tar.gz)

**Comment by huan** [comment]
### Jul 5

[mike-20200705.tar.gz](https://github.com/juzibot/donut-tester/files/4875154/mike-20200705.tar.gz)
